### ISSUE TITLE
link_to helper shows a tags as a string

### ISSUE THREAD

**Issue by merongivian** [joeldrapper/phlex#244]
I've read rails helpers are not complete integrated, is this something that is not yet implemented?

**Comment by joeldrapper** [comment]
That's right, we haven't adapted the `link_to` helper yet. I’m not sure if it’s worth doing since you can just write something like this.

You could always define your own adapter

```ruby
def link_to(...)
  raw helpers.link_to(...)
end
```

Or use `url_for` directly which will allow you to pass the content of the link as a block and also allow to use normal Phlex attributes on the link.

```ruby
def link_to(something, **attributes, &block)
  a(**attributes, href: helpers.url_for(something), &block)
end
```

Now you can use it like this.

```ruby
link_to(@article, class: "bla") { "Hello" }
```
